working-open.md

```diff
@@ -0,0 +1,46 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
+- Not because a "default to open" is an ideal, but because it orients us toward negotiations a set of practices around our shared intentions.
+- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay our our practices of consent and privacy
+
+# How and what?
+
+- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
+
+Legend:
+👀 - View only
+
+**Default open to all**:
+
+- Meeting notes (all hands, working groups) and final documents (handbook), on GitHub and other platforms 👀
+- Aggregated financal statements 👀
+- Project deliverables*, task tracking, meeting notes, and aggregated financial statements 👀
```

Comment:
Is this repetition of "aggregated financial statements" intended to capture the separate org-wide vs project-level scope? If so, perhaps we could make this more clear

Reply:
It is intended to speak to project finances, can you suggest a wording to improve this?

Reply:
- Scope: organization. Aggregate financial statements 👀
- Scope: project. Deliverables*, task tracking, meeting notes, and aggregated financial statements 👀